### PR TITLE
Remove Preview tag from Sql database projects extension

### PR DESCRIPTION
**Pull request by SakshiS-harma**
This PR removes the "Preview" tag from Sql Database projects extension, since it is GAing with this release.

### CODE REVIEW

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -3481,7 +3481,7 @@
 						}
 					],
 					"statistics": [],
-					"flags": "preview"
+					"flags": ""
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:3484]: Don't update insiders here. There should be two separate PRs - one targeting `release/extensions` that updates insiders (since that has the GA version now) and one targeting rc1 that has the stable gallery update.

Comment (SakshiS-harma) [reply extensionsGallery-insider.json]: Sent a PR for the same.

**Comment by Charles-Gagnon** [comment]
@SakshiS-harma You should close this and open a new PR against release/extensions for updating the insiders gallery.
